src/pyats/contrib/plugins/webex.py

```diff
@@ -0,0 +1,170 @@
+import json
+import logging
+import os
+import socket
+
+import requests
+
+from pyats import configuration as cfg
+from pyats.easypy.plugins.bases import BasePlugin
+
+logger = logging.getLogger(__name__)
+
+MESSAGE_URL = 'https://api.ciscospark.com/v1/messages'
+
+MESSAGE_TEMPLATE = """
+## JOB RESULT REPORT
+```
+Job ID        : {job.uid}
+Host          : {job.runtime.env[host][name]}
+Archive       : {job.runtime.runinfo.archive_file}
+Total Tasks   : {job.runtime.tasks.count}
+Total Runtime : {job.elapsedtime}s
+
+Results Summary
+---------------
+Passed        : {job.results[passed]}
+Passx         : {job.results[passx]}
+Failed        : {job.results[failed]}
+Aborted       : {job.results[aborted]}
+Blocked       : {job.results[blocked]}
+Skipped       : {job.results[skipped]}
+Errored       : {job.results[errored]}
+```
+"""
+
+class WebExTeamsNotifyPlugin(BasePlugin):
+    '''
+    Runs after job, sends notification with results to a specified WebEx Teams
```

Comment:
no notification when starting job? if the job in queue on some systems, better to inform when starting the job.